src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -126,11 +136,53 @@ export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
     })
   }
 
+  const { searchCriteriaID } = match?.params
+
+  useEffect(() => {
+    if (!searchCriteriaID) return
+
+    const subscription = fetchQuery<SavedSearchAlertsApp_Alert_Query>(
+      relayEnvironment as Environment,
+      graphql`
+        query SavedSearchAlertsApp_Alert_Query($searchCriteriaID: ID!) {
+          me {
+            savedSearch(id: $searchCriteriaID) {
+              internalID
+              artistIDs
+              userAlertSettings {
+                name
+              }
+            }
+          }
+        }
+      `,
+      { searchCriteriaID: searchCriteriaID }
+    )?.subscribe?.({
```

Comment:
Yeah — I remember wrestling with this in another PR and found that the relay team [discourages](https://relay.dev/docs/api-reference/fetch-query/#behavior-with-topromise) the use of `toPromise()`, primarily because it's not possible to cancel the request as we're doing here by returning a function that calls `.unsubscribe()`.